keras/layers/convolutional.py

```diff
@@ -1449,7 +1464,17 @@ class ZeroPadding2D(Layer):
     # Arguments
         padding: tuple of int (length 2)
             How many zeros to add at the beginning and end of
-            the 2 padding dimensions (axis 3 and 4).
+            the 2 padding dimensions (rows and cols) symmetrically.
+            Setting padding to the values different from (1, 1)
+            overrides top_pad, bottom_pad; and left_pad, right_pad arguments.
```

Comment:
would you like to take a look at what i've done? (attached)
also, i think that you have had a bug on the ordering of the 2D padding class at get_output_shape_for, you mixed up hight/width from rows/cols.

[convolutional.py.zip](https://github.com/fchollet/keras/files/522910/convolutional.py.zip)